README.md

```diff
@@ -3,72 +3,93 @@
 Publisher is the primary content design app for GOV.UK. It provides the user interface for
 entering all the key editorial formats and an API so other apps (primarily frontend) can
 access that data for display. It is intended to work in partnership with Panopticon which
-manages metadata, slugs, titles, etc.
+manages document creation, metadata, slugs, titles, etc. It is sometimes referred to as "mainstream publisher".
 
-## Running in development
+##Screenshots
 
-If you're just interested in running the Publisher locally, with a minimum of interaction
-with other apps, here's how.
-
-### Create a user
+![alt tag](https://raw.githubusercontent.com/alphagov/publisher/update-readme/app/assets/images/publisher_document_screenshot.png)
```

Comment:
You can make these links relative - that also works:

https://raw.githubusercontent.com/alphagov/content-tagger/master/README.md